File: JavaScript/fibonacci.js
const fibonacciSeries = limit => {
  let series = [0, 1];

  while (true) {
    const nextNumber = series[series.length - 1] + series[series.length - 2];

    if (nextNumber > limit) break;

    series.push(nextNumber);
  }
  console.log(series);
};

fibonacciSeries(10);

File: Python/fibonacci.py
def fibonacciSeries(limit):
    series =[0,1]

    while True:
        nextNumber = series[len(series)-1] + series[len(series)-2]

        if nextNumber > limit:
            break
        series.append(nextNumber)

    print(series)
    

fibonacciSeries(10)
    
